webapp/sass/responsive/_mobile.scss

```diff
@@ -1625,6 +1625,11 @@ label span {
     pointer-events: none;
 }
 
+// PLT-5281: link inside a localized message was not clickable because of the rule above
```

Comment:
@aautio the reference to PLT-5281 is not needed, the following is sufficient

```
// link inside a localized message was not clickable because of the rule above
```